.github/workflows/broadcast-frontend-hash_extract-hash-from-release-notes.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+DFX_VERSION="$1" # e.g. 0.14.1
+
+if [[ "$DFX_VERSION" == "latest" ]]
```

Comment:
Another option:
1. check out the sdk repo at the release tag
2. run `shasum -a 256 src/distributed/assetstorage.wasm.gz`

This will work even before the release notes are added to the release tag.